assets/js/builder/Views/SettingsFields.js

```diff
@@ -341,6 +341,10 @@ define( [], function() {
 					defaults.input_type = 'number';
 				break;
 
+				case 'switch-number':
+					defaults.input_type = 'number';
+				break;
+
```

Comment:
Rather than creating a new case that's identical to the `number` case on line 340 we can add `switch-number` to this case.

This would work identically to the `editor` and `switch-editor` cases on 328 & 329

Reply:
got it.